pynetdicom/apps/common.py

```diff
@@ -627,7 +627,7 @@ def handle_store(event, args, app_logger):
             with open(filename, "wb") as f:
                 f.write(b"\x00" * 128)
                 f.write(b"DICM")
-                f.write(write_file_meta_info(f, event.file_meta))
+                write_file_meta_info(f, event.file_meta)
```

Comment:
!

Reply:
Not sure why you think the fix is wrong.

Reply:
Hah, no, my exclamation mark was for the original code. Don't know how I made that mistake